contentcuration/contentcuration/static/js/edit_channel/utils/number_parser.js

```diff
@@ -0,0 +1,111 @@
+/****************
+
+--- NUMBER REGULAR EXPRESSIONS ---
+VALID_NUMBER: [MIXED_NUMBER | FRACTION | DECIMAL | PERCENTAGE | INTEGER]
+DECIMAL: INTEGER.UNFORMATTED_INT
+PERCENTAGE: [DECIMAL | MIXED_NUMBER | FRACTION]%
+MIXED_NUMBER: INTEGER FRACTION
+FRACTION: INTEGER/NON_ZERO_INT
+NON_ZERO_INT: SIGN[NON_ZERO_DIGIT]([DIGIT]{0,2}(,DIGIT{3})*|[UNFORMATTED_INT])?
+INTEGER: SIGN[FORMATTED_INT | UNFORMATTED_INT]
+FORMATTED_INT: [DIGIT]{1,3}[,DIGIT{3}]+
+UNFORMATTED_INT: [DIGIT]*
+DIGIT: [0-9]
+NON_ZERO_DIGIT: [1-9]
+SIGN: -{0,1}
+EXPONENT: [DECIMAL | INTEGER]e+{0,1}[INTEGER]
+
+TODO: Add log and pi?
+
+****************/
+
+const LINE = /^(.*)$/g;
```

Comment:
To answer my own question, we are not.

Should be ok, but see here for which browsers support the `const` keyword: http://caniuse.com/#feat=const